index.html

```diff
@@ -17,13 +17,17 @@
     <h1>Home</h1>
 
     <section>
-      <h2>Hello, I'm Hannah!</h2>
+      <h2>Hello, I'm Hannah and I'm learning to code.</h2>
 
-      <p>This is my first coding task.</p>
+      <p>
+        This website is my first project and I'll be updating it as and when I
+        learn new things.
+      </p>
 
       <img
-        src="https://cdn.dribbble.com/users/2131993/screenshots/15628402/media/7bb0d27e44d8c2eff47276ae86bfd6a3.png?compress=1&resize=400x300&vertical=top"
-        alt="girl working on a laptop"
+        src="./hannah-gorton.png"
+        alt="picture of hannah gorton"
```

Comment:
You don't need to specify that it's a picture.

```suggestion
        alt="Hannah Gorton"
```